cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -328,10 +319,27 @@ pub struct DotFlox {
 }
 
 impl DotFlox {
-    pub fn open(path: impl AsRef<Path>) -> Result<Self, EnvironmentError> {
+    /// Open `<parent_path>/.flox` as a [DotFlox] directory
+    ///
+    /// This method attempts to find a `.flox` directory in the specified parent path,
+    /// and open it as a [DotFlox] directory.
+    /// If the directory is not found, an [EnvironmentError::DotFloxNotFound] is returned.
+    pub fn open_default_in(parent_path: impl AsRef<Path>) -> Result<Self, EnvironmentError> {
```

Comment:
I'm not sure what `default` in this name is supposed to convey.

Reply:
Technically the dot flox (the directory env files are managed in) does not need to be called .flox.
.flox is just the default that is widely implied by the cli.
`open_default_in` thus _opens_ this implied _default_ directory contained _in_ the given directory.

I am aware of the irony DotFlox implying .flox only as a widely used default case🙃

I'm not strongly attached to that name, i actually called it `open_in` before and just added the notion of defaulting to something later, could as well revert that if you think that's clearer.